hbw/app/models/hbw/common/adapter.rb

```diff
@@ -89,6 +85,18 @@ def form(task_id, entity_class)
         end
       end
 
+      def get_forms_for_task_list(tasks)
+        tasks = JSON.parse(tasks)
```

Comment:
Do not overwrite input arguments. You can just do `forms = JSON.parse(tasks).map do |task|` here.